gnovm/pkg/gnolang/static_analysis_test.go

```diff
@@ -0,0 +1,351 @@
+package gnolang
+
+import "testing"
+
+func TestStaticAnalysisShouldPanic(t *testing.T) {
+	defer func() {
+		if r := recover(); r == nil {
+			t.Errorf("The code did not panic")
+		}
+	}()
+
```

Comment:
You should utilize `assert.Panics()`
https://pkg.go.dev/github.com/stretchr/testify/assert#Panics